### ISSUE TITLE
ValueError: expecting a Pin when running the example

### ISSUE THREAD

**Issue by waza-ari** [brainelectronics/micropython-nextion#30]
When running the upload example, I'm getting the following error:

```
>>> import upload
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "upload.py", line 24, in <module>
  File "/lib/nextion/nextion_hardware.py", line 73, in __init__
ValueError: expecting a Pin
```

This should be related to

https://github.com/brainelectronics/micropython-nextion/blob/f46aa184efec1350028c17d7ca70b5c8712c7877/examples/upload/main.py#L19-L23

Which should be

```
import machine

# ...
tx_pin = machine.Pin(21)
rx_pin = machine.Pin(22)

# create Nextion hardware interface
nh = NexHardware(rx_pin=rx_pin, tx_pin=tx_pin)
```

**Comment by brainelectronics** [comment]
I see your point, I initially designed this lib for ESP32 and ESP8266 where an integer is required. I'll update the examples later to make it universal

**Comment by waza-ari** [comment]
Ah that makes a lot of sense. I'm currently running it on a Raspberry Pi Pico using vanilla Micropython (not the Raspberry flavoured one). Let me know if I can be of any help. Thanks so much!

**Comment by brainelectronics** [comment]
Fixed with [0.15.2](https://github.com/brainelectronics/micropython-nextion/releases/tag/0.15.2)
